htdocs/core/lib/company.lib.php

```diff
@@ -710,51 +710,25 @@ function getFormeJuridiqueLabel($code)
  */
 function getCountriesInEEC()
 {
-	global $conf;
-
-	// List of all country codes that are in europe for european vat rules
-	// List found on http://ec.europa.eu/taxation_customs/common/faq/faq_1179_en.htm#9
```

Comment:
Try to keep some comment that can stay helpful after change. For example url here that point to list of countries in EEC.